source/mainnet/smart-contracts/guides/build-schema.rst

```diff
@@ -61,14 +99,20 @@ optional ``parameter`` and ``error`` attributes for the
    #[derive(Serial, Reject, SchemaType)]
    enum InitError { ... }
 
-   #[init(contract = "my_contract", parameter = "InitParameter", error = "InitError")]
+   #[init(contract = "my_contract", parameter = "InitParameter", error = "InitError", event = "Event")]
    fn contract_init<...>(...) -> <..., InitError> { ... }
 
+.. note ::
+
+   The event schema cannot be derived and has to be implemented manually as shown :ref:`in this example <build-event-schema>`.
```

Comment:
I am not sure whether I am misunderstanding here, but in many cases the event schema **can** be derived. It is just that for the CIS2 event it cannot since we use custom tags and don't support like a `tag` attribute.

Reply:
I removed that sentence.